README.md

```diff
@@ -13,9 +13,44 @@ Build and install the provider so that Terraform can use it:
 make install
 ```
 
+## Test
```

Comment:
Please move Test and Debug chapters to be Development subchapters below. Provider will have much more users than developers, so let's keep development-related stuff at the bottom

Reply:
Addressed: 4b1542859c76b33c9d26e41e294fc159f9d6ef26